spec.html

```diff
@@ -4746,7 +4746,7 @@
             1. If the binding for _N_ in _envRec_ is a strict binding, let _S_ be *true*.
             1. If the binding for _N_ in _envRec_ has not yet been initialized, throw a *ReferenceError* exception.
             1. Else if the binding for _N_ in _envRec_ is a mutable binding, change its bound value to _V_.
-            1. Else this must be an attempt to change the value of an immutable binding so if _S_ is *true*, throw a *TypeError* exception.
+            1. Else this must be an attempt to change the value of an immutable binding, so if _S_ is *true*, throw a *TypeError* exception.
```

Comment:
This line is doing too much IMO. I'll probably change it to:

1. Else,
  1. Assert: this is an attempt to change the value of an immutable binding.
  1. If _S_ is *true*, throw a *TypeError* exception.